### PR TITLE
Loosen Redis pin, dep maintenance

### PR DESCRIPTION
**Pull request by nvllsvm**

### CODE REVIEW

File: requires/installation.txt [hunk 1]
```diff
@@ -1,5 +1,5 @@
-consulate>=0.5.1,<2.0
-psycopg2>=2.6,<3.0
+consulate>=0.5.1,<2
+psycopg2>=2.6,<3
 queries>=1.7.3,<2.1
-requests>=2.5.1,<3.0
-redis>=2.0.0,<3.0.0
+requests>=2.5.1,<3
+redis>=2<4
```
Comment (ibnpaul) [inline requires/installation.txt]: Missing a comma here

**Comment by ibnpaul** [comment]
Did you test on Python 2 and 3?

**Comment by nvllsvm** [comment]
@ibnpaul - yep, just retested in both.

**Comment by nvllsvm** [comment]
Interesting discovery - pip doesn't balk at the lack of a comma, but also seems to ignore the `<4` component.

ex. - pip understands that the installed `1.34` is not `>=2`, but then proceeds to install `3.2.1`. All without any warning that the version specifier is partially ignored.

```
$ pip install redis==1.34
$ pip install 'redis>=2<3'
```

**Pull request merged**